hydrus/data/crud.py

```diff
@@ -56,6 +56,10 @@ def insert(object_, id_=None):
     """Insert an object to database [POST] and returns the inserted object."""
     # NOTE: We are inserting the object, no need to check if similar one already exists.
     #       Data can be redundant/identical, they must have different "@id"
+

```

Comment:
You can't insert any other type of subsystem. That is because the other classes are not yet in the Database. They still need to be added.
And we still need to decide how we want to automatically add them
I added thermal manually. I forgot to dereference it. But for now you will have to add all classes and properties to RDFClass and BaseProperty manually. I have inserted the properties used by the random generated data. But there are others that need to be added
There is no issue with the insert method, only that the relevant classes are not present in the DB

Also, about the type_ attribute, I was wondering if it is needed at all since we can add a property called type and store that in graph itself. For now I will add the type and the classes. I will dereference as well.